web_monitoring/db.py

```diff
@@ -142,13 +143,17 @@ class method :meth:`Client.from_env`.
     url : string, optional
         Default is ``https://api.monitoring.envirodatagov.org``.
     """
-    def __init__(self, email, password, url=DEFAULT_URL):
+    def __init__(self, email, password, url=DEFAULT_URL, timeout=DEFAULT_TIMEOUT):
```

Comment:
Make sure to add this to the class docstring above, too!

Reply:
Good catch